collectors/0/old_flume_jmx.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python
+
+import sys
+from flume_jmx import FlumeJmxMonitor
+
+class OldFlumeJmxMonitor(FlumeJmxMonitor):
```

Comment:
no no, we should be able to target old flume just the to existing webtier and the new one just to flafka, if that's what you mean